autoload/neomake/makers/ft/erlang.vim

```diff
@@ -5,8 +5,45 @@ endfunction
 
 function! neomake#makers#ft#erlang#erlc() abort
     return {
+        \ 'exe': 'erlc',
+        \ 'args': function('neomake#makers#ft#erlang#GlobPaths'),
         \ 'errorformat':
             \ '%W%f:%l: Warning: %m,' .
             \ '%E%f:%l: %m'
         \ }
 endfunction
+
+function! neomake#makers#ft#erlang#GlobPaths() abort
+    " Pick the rebar3 profile to use.
+    if match(expand('%'), 'SUITE.erl$') > -1
+        let profile = 'test'
+    else
+        let profile = 'default'
+    endif
+    " Find project root directory.
+    let root = fnamemodify(neomake#utils#FindGlobFile('rebar.config'), ':h')
```

Comment:
This should handle the case where `rebar.config` is not found, i.e. `neomake#utils#FindGlobFile('rebar.config')` being empty.
This should be done in the beginning already.